contracts/modules/staking/StakingPool.sol

```diff
@@ -1357,6 +1351,21 @@ contract StakingPool is IStakingPool, Multicall {
     emit PoolFeeChanged(msg.sender, newFee);
   }
 
+  /// @notice Rolls back lastAccNxmUpdate if it is lower than the current block timestamp to avoid underflow
+  function rollbackLastAccNxmUpdate() external {
+    if (poolId != 8) {
+      return;
+    }
+
+    uint bucketStartTime = (firstActiveBucketId + 1) * BUCKET_DURATION;
+    uint trancheEndTime = (firstActiveTrancheId + 1) * TRANCHE_DURATION;
+    uint lowerTimeStamp = Math.min(bucketStartTime, trancheEndTime);
+
+    if (lowerTimeStamp < lastAccNxmUpdate) {
+      lastAccNxmUpdate = lowerTimeStamp.toUint32();
+    }
```

Comment:
How about this:
```
    if (lowerTimeStamp >= lastAccNxmUpdate) { // flipped the condition
      revert("Not applicable");
    }

    lastAccNxmUpdate = lowerTimeStamp.toUint32();
```
+ also revert if the `poolId != 8`. Reasoning would be to prevent the caller from sending a tx if it's a no-op.